atlasdb-jepsen-tests/src/jepsen/atlasdb/timelock.clj

```diff
@@ -2,7 +2,19 @@
   (:require [clojure.tools.logging :refer :all]
             [jepsen.control :as c]
             [jepsen.db :as db]
-            [jepsen.os.debian :as debian]))
+            [jepsen.nemesis :as nemesis]
+            [jepsen.os.debian :as debian])
+  (:import com.palantir.atlasdb.http.TimelockUtils))
+
+(defn start!
+  "Starts timelock"
+  [node]
+  (c/su
+    (info node "Starting timelock server")
+    (c/exec :env "JAVA_HOME=/usr/lib/jvm/java-8-oracle" "/timelock-server/service/bin/init.sh" "start")
+    (info node "Waiting until timelock node is ready")
+    (TimelockUtils/waitUntilHostReady (name node))
```

Comment:
I don't think we should have this `start!` function in. We explicitly changed the test to _not_ wait until the server had started, and instead already serve requests while it is starting up, to make this more real-world.